ivy/functional/frontends/tensorflow/func_wrapper.py

```diff
@@ -92,3 +92,35 @@ def new_fn(*args, **kwargs):
 
 def to_ivy_arrays_and_back(fn: Callable) -> Callable:
     return outputs_to_tensorflow_array(inputs_to_ivy_arrays(fn))
+
+
+def map_raw_ops_alias(alias: callable, **function_kwargs):
```

Comment:
We shouldn't accept `**function_kwargs` here, we could instead accept a dictionary by some name say `kwargs_to_replace` where every pair in the dictionary is a mapping from the old name of the keyword argument and the new name of the keyword argument when you call the function, e.g. `{"dimension": "axis"}` for the argmax special case you've mentioned below. This way we could handle all edge cases, whilst also making sure that we only wrap once during the import.